src/test/java/com/jcabi/w3c/DefaultHtmlValidatorTest.java

```diff
@@ -91,13 +114,13 @@ public void validatesHtmlDocument() throws Exception {
     public void throwsIOExceptionWhenValidationServerErrorOccurred()
         throws Exception {
         final Set<Integer> responses = new HashSet<Integer>(
-            Arrays.asList(
-                HttpURLConnection.HTTP_INTERNAL_ERROR,
-                HttpURLConnection.HTTP_NOT_IMPLEMENTED,
-                HttpURLConnection.HTTP_BAD_GATEWAY,
-                HttpURLConnection.HTTP_UNAVAILABLE,
-                HttpURLConnection.HTTP_GATEWAY_TIMEOUT,
-                HttpURLConnection.HTTP_VERSION
+                        Arrays.asList(
+                                      HttpURLConnection.HTTP_INTERNAL_ERROR,
```

Comment:
@jrdalpra lets keep the indentation here and below unchanged
we use 4 spaces usually